lib/full-site-editing/edit-site-page.php

```diff
@@ -76,6 +76,35 @@ function gutenberg_get_editor_styles() {
 	return $styles;
 }
 
+/**
```

Comment:
Can we put the PHP into `lib/compat`? `wordpress-5.9` if we know it will be backported or `wordpress-6.0` otherwise.

Reply:
I think this should be internal function/logic that lives in the `site-editor.php` file, but we can use compat dir for the plugin.

Reply:
Done :) I'm not sure if it's the correct file, but I couldn't find anything specific for the site editor.